.ado/windows-vs-pr.yml

```diff
@@ -725,3 +739,7 @@ jobs:
     parameters:
       name: IntegrationTest
       BuildPlatform: x64
+
+  - template: templates/macos-tests-job.yml
```

Comment:
So, this is a little weird being in a file called `windows-vs-pr`. I looked at renaming the file, or adding this as its own YAML, but we need to do some reconfiguration if we rename this, and having as a separate file would mean we can't gate with the same `Setup` job we have right now.